docs/contributing/publishing-a-pre-release.md

```diff
@@ -16,8 +16,17 @@ No changes get published to npm as part of the process.
 
 4. Run `npm run release-to-branch`.
 
-There should now be a branch `pre-release-[your-branch-name-here]` pushed to remote, which you can install with:
+There should now be a branch `pre-release-[your-branch-name-here]` pushed to remote.
+
+5. In GOVUK Design System:
```

Comment:
I think this is useful, but I think there are use cases for pre-releases that _do not_ involve a change to the Design System, and this change will be confusing for those users.

For example, if a team wanted us to make a change to an existing component and we wanted to be able to share a pre-release with them so they could test it, they would install the pre-release branch in their own application, not the Design System.

I suggest making it clear that this step is optional, splitting it out entirely or keeping it generic.

(Minor: GOV.UK, not GOVUK)